docs/src/development.md

```diff
@@ -0,0 +1,20 @@
+# Snowflake development
+
+!!! warning
+    This page is under construction 🚧
+
+## Run coverage locally
+
```

Comment:
I had to run Pkg.Instantiate() to get the coverage report to run... might worth mentioning something like
```suggestion
If you haven't already, instantiate the project with Julia's package manager.
```bash
julia --project=. -e 'using Pkg; Pkg.Instantiate()'
``` .
```
But please rewrite to more canonical Julia-isms if you can :sweat_smile: